app/views/traces/_trace.html.erb

```diff
@@ -1,35 +1,44 @@
-<tr>
-  <td>
-    <% if Settings.status != "gpx_offline" %>
-      <% if trace.inserted %>
-        <a href="<%= url_for :controller => "traces", :action => "show", :id => trace.id, :display_name => trace.user.display_name %>"><img src="<%= url_for :controller => "traces", :action => "icon", :id => trace.id, :display_name => trace.user.display_name %>" border="0" alt="" /></a>
-      <% else %>
-        <span class="text-danger"><%= t ".pending" %></span>
-      <% end %>
+<li class="media mb-1 p-2 <%= cycle("", " bg-light") %>">
```

Comment:
We recently moved away from `cycle` in templates, although this is less intrusive than what we had previously. See https://github.com/openstreetmap/openstreetmap-website/commit/5fdada204cf92d8ca99300927713724aaaf1b2bd for the change.

This proposes changing the colour for the stripes, which makes it inconsistent with the other striped tables/lists e.g. messages.

Is there a different way to achieve this? Or alternatively, does writing more CSS to support attempting to stripe a list suggest that we should be sticking with a table?